src/vivarium/framework/results/manager.py

```diff
@@ -55,9 +56,9 @@ def get_results(self) -> dict[str, pd.DataFrame]:
         Notes
         -----
         self._results_context.observations is a list where each item is a dictionary
-        of the form {lifecycle_phase: {(pop_filter, stratification_names): List[Observation]}}.
+        of the form {lifecycle_phase: {(PopulationFilterDetails, stratification_names): List[Observation]}}.
```

Comment:
This is actually not true. I missed fixing this docstring when I redid the results stuff this summer. `results_context.observations` is a `dict[str, Observation]`.